all-actors/src/main/java/org/sunbird/CertMapper.java

```diff
@@ -79,6 +82,14 @@ private Issuer getIssuer(Map<String, Object> issuerData) {
         return issuer;
     }
 
+    private Criteria getCriteria(Map<String, Object> criteriaData) {
+        Criteria criteria = new Criteria();
```

Comment:
can it be simpler than this?
return new objectMapper().convertValue(criteriaData, Criteria.class);

Reply:
done